.yarnrc

```diff
@@ -1 +1 @@
-ignore-engines true
+ignore-engines false
```

Comment:
There was a reason this was done (I'm not remembering right now), we'll need some more testing before a merge

Reply:
The `.yarnrc` hasn't been modified since the switch from npm to yarn in 2017. CI is passing and this works locally, but I'm wondering if the file is relevant anywhere else.

Reply:
yeah but we tried removing it at one point and failed, trying to rack my brain

Reply:
The only thing I can think of is if `vets-website` is cloned in another repo's CI process, and that repo is using a different version of `node`. I think the only repo that does that is `content-build`, and that uses the same version of `node`. This might have been more of an issue back when we were using Jenkins, maybe?